stat.c

```diff
@@ -306,7 +345,7 @@ void show_group_stats(struct group_run_stats *rs, struct buf_output *out)
 		max = num2str(rs->max_bw[i], rs->sig_figs, 1, i2p, rs->unit_base);
 		maxalt = num2str(rs->max_bw[i], rs->sig_figs, 1, !i2p, rs->unit_base);
 		log_buf(out, "%s: bw=%s (%s), %s-%s (%s-%s), io=%s (%s), run=%llu-%llumsec\n",
-				rs->unified_rw_rep ? "  MIXED" : str[i],
+				(rs->unified_rw_rep == 1) ? "  MIXED" : str[i],
```

Comment:
Not specific to this location, but in general - do we want to make these (now) three states specific by using some kind of enum/deifne for them? == 1 or == 2 means nothing when you are reading the code, but symbolics would help with that.

Might even then make sense to turn the option into a posval instead, and use the same values there.